charts/monitoring-config/dashboards/graphql-stats.json

```diff
@@ -118,7 +118,98 @@
           "refId": "A"
         }
       ],
-      "title": "10 Slowest document types (Last 6 hours)",
+      "title": "10 Slowest Content Store document types (Last 6 hours)",
+      "type": "table"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "prometheus"
+      },
+      "description": "10 slowest average response times to content store, broken up by document type",
```

Comment:
Description and title 8 lines above need changing, both refer to content store.

Reply:
oops, good spot. Fixed!

Reply:
The title you can see above is for the other panel, so that ones right

Reply:
Yep my bad, silly Json ordering